extensions/amazon-aws/src/cloudformation.tsx

```diff
@@ -1,61 +1,33 @@
 import { getPreferenceValues, ActionPanel, List, Detail, Action } from "@raycast/api";
-import { useState, useEffect } from "react";
 import * as AWS from "aws-sdk";
 import { Preferences } from "./types";
 import { StackSummary } from "aws-sdk/clients/cloudformation";
 import setupAws from "./util/setupAws";
+import { useCachedPromise } from "@raycast/utils";
 
 setupAws();
+const cloudformation = new AWS.CloudFormation({ apiVersion: "2016-11-15" });
 
 export default function ListStacks() {
-  const preferences: Preferences = getPreferenceValues();
-  const cloudformation = new AWS.CloudFormation({ apiVersion: "2016-11-15" });
+  const { data: stacks, error, isLoading } = useCachedPromise(fetchStacks);
```

Comment:
I'm not sure that useCachedPromise is okay here, because if it is storing result between command runs, it means that if `stacks` will be changed in AWS, it will be not refetched because it will get data from catch. So maybe it makes sense to use `usePromise` instead

<img width="787" alt="Screenshot 2022-11-08 at 15 17 11" src="https://user-images.githubusercontent.com/13422799/200574802-4d4b5dd4-293d-4fb1-8982-e625cf20d2e6.png">


Did you test that case?

Reply:
or maybe I just do not understand how it works

if it is refetching on next command run but using cached value as initial value, then it is okay to use this one

Reply:
> if it is refetching on next command run but using cached value as initial value, then it is okay to use useCachedPromise

that was my assumption. I've confirmed that and it **does** work like that. so using `useCachedPromise` is the best choice here IMO.

Reply:
Yep, I agree, cool 🚀